content-tool/tool/src/webapp/vm/resources/sakai_replace_file.vm

```diff
@@ -29,7 +29,7 @@
 					<label>$tlang.getString("label.name")</label>
 				</td>
 				<td style="width:180px;padding-left:0" id="itemName">
-					$validator.escapeHtml($item.name)
+					$item.getOriginalFileName()
```

Comment:
The pattern in velocity is to use `$item.originalFileName` rather than explicitly calling the Java method.